src/main/scala/mesosphere/marathon/scheduling/LegacyScheduler.scala

```diff
@@ -0,0 +1,45 @@
+package mesosphere.marathon
+package scheduling
+
+import akka.Done
+import mesosphere.marathon.core.instance.{Goal, Instance}
+import mesosphere.marathon.core.launcher.OfferProcessor
+import mesosphere.marathon.core.task.tracker.InstanceTracker
+import mesosphere.marathon.core.task.update.TaskStatusUpdateProcessor
+import mesosphere.marathon.state.PathId
+import org.apache.mesos.Protos
+
+import scala.async.Async.{async, await}
+import scala.concurrent.{ExecutionContext, Future}
+
+case class LegacyScheduler(
+    offerProcessor: OfferProcessor,
+    instanceTracker: InstanceTracker,
+    statusUpdateProcessor: TaskStatusUpdateProcessor) extends Scheduler {
+
+  //override def create(runSpec: RunSpec): Future[Instance.Id]
```

Comment:
This is an interesting one. The current setup uses `LaunchQueue.add(runSpec, numberOfInstances)`. I hesitate to use that as is.